From b0154ff51bef4913d6b0ddf2ba4151dfb66dda57 Mon Sep 17 00:00:00 2001
From: Boris Itkis <borisi@interfacemasters.com>
Date: Sun, 2 Nov 2014 00:13:09 -0700
Subject: [PATCH] Make getattr return false if no attribute is available

---
 python_jsonschema_objects/classbuilder.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python_jsonschema_objects/classbuilder.py b/python_jsonschema_objects/classbuilder.py
index 37d885c..c542060 100644
--- a/python_jsonschema_objects/classbuilder.py
+++ b/python_jsonschema_objects/classbuilder.py
@@ -566,7 +566,7 @@ def make_property(prop, info, desc=""):
                 if isinstance(val, typ):
                     ok = True
                     break
-                elif getattr(typ, 'isLiteralClass'):
+                elif getattr(typ, 'isLiteralClass', False):
                     try:
                         val = typ(val)
                     except Exception as e:
-- 
1.9.3 (Apple Git-50)

